eng/pipelines/runtime-official.yml

```diff
@@ -41,6 +41,11 @@ stages:
 - stage: Build
   jobs:
 
+  #
+  # Source Index Build
+  #
+  - template: ./source-index.yml
```

Comment:
do we need it in both the official build and CI?

Reply:
If we don't do it in CI breaks will get checked in that stop the index from updating.

Reply:
The official build is the one that matters though. It will do the uploading, the CI one is just for preventing breaks.

Reply:
I see, ok. We can't do the upload from CI? i.e, when not in PRs?

Reply:
The upload has to happen in the internal project, because it needs a secret.

Reply:
I imagined that but didn't hurt to ask.